tests/functional/generate_pages.py

```diff
@@ -813,9 +813,16 @@ def process_block(
                 process_guidance(context, page_spec)
 
         elif block["type"] == "CalculatedSummary":
-            process_calculated_summary(
-                block["calculation"]["answers_to_calculate"], page_spec
-            )
+            if block["calculation"].get("answers_to_calculate"):
+                process_calculated_summary(
+                    block["calculation"]["answers_to_calculate"], page_spec
+                )
+            else:
+                answer_ids_for_block = [
+                    value["identifier"]
+                    for value in block["calculation"]["operation"]["+"]
```

Comment:
Same here about, not relying on first level of operator.

Reply:
Updated this 👍